circle.yml

```diff
@@ -8,8 +8,7 @@ machine:
 
 dependencies:
   pre:
-    # should just set env vars in circleci
-    - touch .env
+    - cp .env-dist .env
```

Comment:
Now I'm confused. By GitHub. 
[That line is already in there](https://github.com/mozilla/tecken/blob/master/circle.yml)